src/main/java/core/basesyntax/model/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Square extends Color {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
```

Comment:
The method `getArea` correctly calculates the area of the square, which aligns with the single responsibility principle.